packages/browser/src/core/analytics/index.ts

```diff
@@ -201,7 +205,13 @@ export class Analytics
     super()
     const cookieOptions = options?.cookie
     const disablePersistance = options?.disableClientPersistence ?? false
-    this.settings = new AnalyticsInstanceSettings(settings)
+
+    // Extract apiHost from Segment.io settings if available
+    const segmentLoadOptions = options?.integrations?.['Segment.io'] as
+      | SegmentioSettings
+      | undefined
+    const apiHost = segmentLoadOptions?.apiHost
```

Comment:
This isn't the true value -- these loadOptions are just overrides -- we would need to take into account the cdnSettings apiHost value. 

Clean option is to create a metadata? Object on the Segmentio plugin itself with the apiHost, and then having a getter here?